src/sql/workbench/browser/modelComponents/media/dropdown.css

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.dropdown-error-container {
```

Comment:
Rules for components should always be scoped to the components themselves, e.g.

`modelview-dropdown .dropdown-error-container`

(see other css files in this folder for examples)